src/pages/home/report/ReportActionCompose.js

```diff
@@ -45,6 +45,7 @@ class ReportActionCompose extends React.Component {
         this.state = {
             isFocused: false,
             textInputShouldClear: false,
+            isCommentEmpty: this.props.comment.length === 0,
```

Comment:
This should also be changed to `props.comment.length === 0`.

Reply:
No need to reference `this` inside the `constructor` thanks.

Reply:
Done @marcaaron